docs/api/structures/base-window-options.md

```diff
@@ -97,9 +97,9 @@
   * `height` Integer (optional) - The height of the title bar and Window Controls Overlay in pixels. Default is system height.
 * `trafficLightPosition` [Point](point.md) (optional) _macOS_ -
   Set a custom position for the traffic light buttons in frameless windows.
-* `roundedCorners` boolean (optional) _macOS_ - Whether frameless window
-  should have rounded corners on macOS. Default is `true`. Setting this property
-  to `false` will prevent the window from being fullscreenable.
+* `roundedCorners` boolean (optional) _macOS_ _Windows_ - Whether frameless window
```

Comment:
Do you think it'd be worth mentioning the Windows 11 requirement? Looks like Windows 10 end of support is scheduled for October 14, 2025.

Reply:
done!